pyuvdata/uvdata/tests/test_mir.py

```diff
@@ -13,23 +13,53 @@
 
 import pytest
 import numpy as np
+import pyuvdata.tests as uvtest
```

Comment:
Looks like both relative and absolute imports from other pyuvdata modules are used in the same file.  General wisdom says that files that are part of a python package should use relative imports so you can guarantee you don't import some other thing that happens to have the same name.

Reply:
Fixed in 8f8a8af4.